app/bundles/CoreBundle/Token/TokenAttribute.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * @copyright   2016 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Token;
+
+/**
+ * CLass Match.
+ */
+class TokenAttribute
+{
+    private $attribute;
+
+    /**
+     * Token constructor.
```

Comment:
This information also has no value. I know that PHP Storm is doing this, but it's waste of space :)